### DISCUSSION TITLE
Dual WAN ddns

### DISCUSSION THREAD

**Discussion by curvingedge** [willswire/unifi-ddns#58] [Q&A] [answered]
I have two ISP connections and Att just raised my static ip by $15, so now I want to cancel the static ip. I currently use dyndns to update my cloudflare subdomain for the failover wan. iirc Unifi will not let me use another dyndns for the primary wan. Is there already a workaround for this or could another ddns provider be selected?

**Comment by TanerH** [comment, marked as answer]
Did you ever solve this?

Once I set my primary WAN to "custom", I can't use "custom" for my failover WAN.  Or can I just use a different one, like "dyndns" (but still use it for Cloudflare?)

**Reply by curvingedge** [reply]
Thanks for the response. Unifi added Dyn and DynDNS so I was able to add both WAN connections.

**Comment by willswire** [comment]
I’ll create some documentation around this in a new FAQ resource; you can choose the dyndns provider in addition to the custom provider for secondary WAN connections.
